db/migrate/20161209113503_add_timestamps_to_bookable_slots.rb

```diff
@@ -0,0 +1,15 @@
+class AddTimestampsToBookableSlots < ActiveRecord::Migration[5.0]
+  class BookableSlot < ApplicationRecord; end
+
+  def change
+    add_timestamps :bookable_slots, null: true
+
+    BookableSlot.update_all(
+      created_at: Time.zone.now,
+      updated_at: Time.zone.now
+    )
```

Comment:
I dunno it's just something I always think about when doing these timestamps migrations. Like if I wanted to find which ones were created by migration I would do something like `WHERE created_at == updated_at`.

Probably a dumb idea though. Ignore it...